packages/bloc_test/lib/src/bloc_test.dart

```diff
@@ -110,7 +110,7 @@ import 'package:test/test.dart' as test;
 ///   act: (bloc) => bloc.add(CounterEvent.increment),
 ///   expect: () => [1],
 ///   verify: (_) {
-///     verify(() => repository.someMethod(any())).called(1);
+///     verify(() => repository.someMethod(any()).called(1));
```

Comment:
I believe the original snippet is correct. `called` is invoked on the result of `verify` as per the `mocktail` API. Closing for now but if you feel this is incorrect please feel free to let me know, thanks!